modules/ivts/dev.galasa.ivts/settings.gradle

```diff
@@ -0,0 +1,20 @@
+
+
+
+// Tell gradle where it should look to find the plugins and dependencies it needs to build.
+pluginManagement {
+    repositories {
+        mavenLocal()
+        // To use the bleeding edge version of galasa's obr plugin, use the development obr
+        maven {
+           url 'https://development.galasa.dev/main/maven-repo/obr'
```

Comment:
This will break on a fork ?

Reply:
Hi Mike, I'm not 100% sure what you mean here, can you clarify please?

Reply:
I mean if I fork the galasa code, I want to be building against that, rather than what's stood up on some galasa infrastructure ?

Reply:
I guess at that point a forked build would still be referring to mavenLocal() when it runs inside gh actions.

Reply:
This value is now `$sourceMaven` and provided to the gradle file by the build process calling it (either locally which supplies this URL or GitHu which supplies the artifacts download directory)